obspy/taup/__init__.py

```diff
@@ -140,50 +140,106 @@
 --------
 
 If ray paths have been calculated, they can be plotted using the
-:meth:`~obspy.taup.tau.Arrivals.plot` method:
+:meth:`~obspy.taup.tau.Arrivals.plot_rays` method:
 
 >>> arrivals = model.get_ray_paths(source_depth_in_km=500,
 ...                                distance_in_degree=130)
->>> arrivals.plot()  # doctest: +SKIP
+>>> arrivals.plot_rays()  # doctest: +SKIP
```

Comment:
Do these plots take a lot of time? In principle we should skip as few commands as possible.. any big downsides of having these doctests (here and below) executed?